docs/xsl/common.xsl

```diff
@@ -54,19 +54,10 @@
     <script type='text/javascript' src='js/jquery/jquery-1.11.0.js'></script>
     <script type='text/javascript' src='js/jquery/jquery-migrate-1.2.1.min.js'></script>
 
-    <!--[if lte IE 8]>
-    <link rel="stylesheet" href="avada-stub/css/ie8.css" />
-    <![endif]-->
+    <link rel='stylesheet' id='inline-blob-janusgraph-docs-specific' href='css/docs.css' type='text/css' media='all' />
 
-    <!--[if IE]>
-    <link rel="stylesheet" href="avada-stub/css/ie.css" />
-    <![endif]-->
-
-    <link rel='stylesheet' id='style-css-css'  href='avada-stub/style.css' type='text/css' media='all' />
-    <link rel='stylesheet' id='media-css-css'  href='avada-stub/css/media.css' type='text/css' media='all' />
-    <link rel='stylesheet' id='animate-css-css'  href='avada-stub/css/animate-custom.css' type='text/css' media='all' />
-    <link rel='stylesheet' id='inline-blob-from-avada-prototype' href='avada-stub/css/inlineblob.css' type='text/css' media='all' />
-    <link rel='stylesheet' id='inline-blob-from-avada-prototype-janusgraph-docs-specific' href='css/docs.css' type='text/css' media='all' />
+    <link rel='icon' type='image/jpeg' href='images/janusgraph-logomark.png' />
+    <link rel='apple-touch-icon' type='image/jpeg' href='images/janusgraph-logomark.jpg' />
```

Comment:
It would be better to use PNG than JPG for logos as PNGs are not lossy and don't have funny artifacts due to JPG compression. We have plenty of high-resolution PNGs — does this feature not accept PNGs?